src/components/usersettings/usersettingsbuilder.js

```diff
@@ -1,25 +1,23 @@
-define(['appSettings', 'events'], function (appsettings, events) {
+define(['appSettings', 'events'], function (appSettings, events) {
     'use strict';
 
     function onSaveTimeout() {
-
-        var self = this;
-        self.saveTimeout = null;
-        self.currentApiClient.updateDisplayPreferences('usersettings', self.displayPrefs, self.currentUserId, 'emby');
+        this.saveTimeout = null;
```

Comment:
why? 
`self `is a normal variable and accessible inside the callback. This also has the advantage that you can access the `this `value of the callback itself